### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
Isolated reusable components from every page.
- [x] Navbar
- [x] SideBar
- [x] NoteContainer
- [x] Note Card

### CODE REVIEW

File: src/pages/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  flex-direction: column;
+}
+.sidebar-icon {
+  font-size: 1.8rem;
+}
+.sidebar-drawers {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  /* margin-top: 1rem; */
```
Comment (Aazam10) [inline src/pages/Home/Home.css]: You can remove commented code if not necessary

**Review by Aazam10 — APPROVED** [review]
Everything is great kevin.Just a suggestion you can use variables instead of using color directly.

File: src/pages/Landing/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { heroImg } from '../../assests';
+import './Landing.css';
+export const Landing = () => {
+  return (
+    <main class="center">
+      <section class="wrapper-center">
```
Comment (Tejinder15) [inline src/pages/Landing/Landing.jsx]: Hey @Kevin-Solomon  just change class attribute to className .

File: src/pages/Landing/Landing.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            <h3>Meet your modern</h3>
+            <h3 class="text-muted">Note Taking App</h3>
+            <p class="hero-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Voluptatibus totam est laudantium voluptatum voluptatem asperiores
+              veritatis quidem inventore voluptate aliquid.
+            </p>
+          </div>
+          <button class="btn primary-btn">
+            <a href="/home.html">Join Now</a>
```
Comment (Tejinder15) [inline src/pages/Landing/Landing.jsx]: Since you have already included routes you can also use Link instead of a tag.

**Review by Tejinder15 — APPROVED** [review]

**Pull request merged**